### PR TITLE
data: roberanegussie

### PR DESCRIPTION
**Pull request by roberanegussie**
## Changes proposed
Adding my profile

## Check List (Check all the applicable boxes) 

- [ ] My code follows the code style of this project.
- [ ] My change requires changes to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] All new and existing tests passed.
- [ ] This PR does not contain plagiarized content.
- [ ] The title of my pull request is a short description of the requested changes.

## Note to reviewers

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/5689"></a>

### CODE REVIEW

File: data/roberanegussie.json [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+  "links": [
+    {
+      "name": "My Twitter Handle",
+      "url": "https://twitter.com/roberanegussie",
+      "icon": "FaTwitter"
+    },
+    {
+      "name": "My LinkedIn Profile",
+      "url": "https://www.linkedin.com/in/roberatoye/",
+      "icon": "FaLinkedIn"
```
Comment (khushalarora) [inline data/roberanegussie.json:14]: Please replace it with **' FaLinkedin '**  :+1: 
You can also checkout [ICONS](https://linkfree.eddiehub.io/icons) for valid icons.
```Suggestion
      "icon": "FaLinkedin"
```

File: data/roberanegussie.json [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+    },
+    {
+      "name": "My GitHub Account",
+      "url": "https://github.com/roberanegussie",
+      "icon": "FaGitHub"
```
Comment (khushalarora) [inline data/roberanegussie.json:19]: Please replace it with **' FaGithub '**  :+1: 
```Suggestion
      "icon": "FaGithub"
```

**Review by khushalarora — CHANGES_REQUESTED** [review]
Hi @roberanegussie, thank you for contributing :tada:

**Comment by roberanegussie** [comment]
@khushalarora  I have made the changes kindly review.

**Review by kumarsonsoff3 — APPROVED** [review]
Looks good, Thanks for your Contribution! 🎉 

<p align="center"></p>

**Pull request merged**